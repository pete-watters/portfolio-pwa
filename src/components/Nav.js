import React from 'react';
import { NavLink } from 'react-router-dom';
import Icon from './Icon';
import CONFIG from '../constants';

const Nav = () =>
    <nav>
      <NavLink to={CONFIG.ROUTES.ABOUT}>About</NavLink>
      <NavLink to={CONFIG.ROUTES.BLOG}>Blog</NavLink>
      <Icon icon="twitter" href={CONFIG.SOCIAL.TWITTER} />
      <Icon icon="github" href={CONFIG.SOCIAL.GITHUB} />
      <Icon icon="linkedIn" href={CONFIG.SOCIAL.LINKEDIN} />
      <Icon icon="stackOverflow" href={CONFIG.SOCIAL.STACKOVERFLOW} />
      <Icon icon="medium" href={CONFIG.SOCIAL.MEDIUM} />
      <Icon icon="instagram" href={CONFIG.SOCIAL.INSTAGRAM} />
    </nav>;

export default Nav;