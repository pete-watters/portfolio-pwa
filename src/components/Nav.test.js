import React from 'react';
import { shallow } from 'enzyme';
import Nav from './Nav';

describe('Nav', () => {
  it('should render the component correctly', () => {
    expect(shallow(<Nav />)).toMatchSnapshot();
  });
});
