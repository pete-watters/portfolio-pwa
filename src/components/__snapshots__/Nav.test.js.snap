// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should render the component correctly 1`] = `
<nav>
  <NavLink
    to="/about"
  >
    About
  </NavLink>
  <NavLink
    to="/blog"
  >
    Blog
  </NavLink>
  <SocialMedia />
</nav>
`;
